project/Win32BuildSetup/AppxManifest.xml.in

```diff
@@ -23,10 +23,181 @@
+              <uap:FileType>.m2ts</uap:FileType>
+              <uap:FileType>.m3u</uap:FileType>
+              <uap:FileType>.m3u8</uap:FileType>
+              <uap:FileType>.m4a</uap:FileType>
+              <uap:FileType>.mac</uap:FileType>
+              <uap:FileType>.mk3d</uap:FileType>
+              <uap:FileType>.mka</uap:FileType>
+              <uap:FileType>.mkv</uap:FileType>
+              <uap:FileType>.mov</uap:FileType>
+              <uap:FileType>.mp2</uap:FileType>
+                <uap:FileType>.mp2v</uap:FileType>
+              <uap:FileType>.mp4</uap:FileType>
+              <uap:FileType>.m4v</uap:FileType>
+              <uap:FileType>.mp4v</uap:FileType>
+              <uap:FileType>.mp+</uap:FileType>
+                <uap:FileType>.mlp</uap:FileType>
+              <uap:FileType>.mp3</uap:FileType>
+              <uap:FileType>.mpa</uap:FileType>
+              <uap:FileType>.mpc</uap:FileType>
+              <uap:FileType>.mpd</uap:FileType>
+              <uap:FileType>.mpdsp</uap:FileType>
+                <uap:FileType>.mpe</uap:FileType>
+              <uap:FileType>.mpeg</uap:FileType>
+                <uap:FileType>.mpeg1</uap:FileType>
+                <uap:FileType>.mpeg2</uap:FileType>
+                <uap:FileType>.mpeg4</uap:FileType>
+              <uap:FileType>.mpg</uap:FileType>
+              <uap:FileType>.mpls</uap:FileType>
+              <uap:FileType>.mpp</uap:FileType>
+              <uap:FileType>.mpt</uap:FileType>
+                <uap:FileType>.mpv2</uap:FileType>
+              <uap:FileType>.mss</uap:FileType>
+              <uap:FileType>.mts</uap:FileType>
+                <uap:FileType>.mtv</uap:FileType>
+                <uap:FileType>.mxf</uap:FileType>
+                <uap:FileType>.mxg</uap:FileType>
+              <uap:FileType>.nrg</uap:FileType>
+              <uap:FileType>.nsv</uap:FileType>
+                <uap:FileType>.nut</uap:FileType>
+              <uap:FileType>.nuv</uap:FileType>
+              <uap:FileType>.oga</uap:FileType>
+              <uap:FileType>.ogg</uap:FileType>
+              <uap:FileType>.ogm</uap:FileType>
+              <uap:FileType>.ogv</uap:FileType>
+                <uap:FileType>.ogx</uap:FileType>
+              <uap:FileType>.opus</uap:FileType>
+              <uap:FileType>.pls</uap:FileType>
+                <uap:FileType>.ps</uap:FileType>
+              <uap:FileType>.pxml</uap:FileType>
+              <uap:FileType>.rcv</uap:FileType>
+              <uap:FileType>.rec</uap:FileType>
+              <uap:FileType>.rm</uap:FileType>
+              <uap:FileType>.rma</uap:FileType>
+              <uap:FileType>.rmt</uap:FileType>
+              <uap:FileType>.rmvb</uap:FileType>
+              <uap:FileType>.rsd</uap:FileType>
+              <uap:FileType>.rss</uap:FileType>
+              <uap:FileType>.pva</uap:FileType>
+              <uap:FileType>.qt</uap:FileType>
+              <uap:FileType>.sap</uap:FileType>
```

Comment:
it was sap:// protocol from removed from our vfs. no idea what .sap extension is.